src/helpers/fetch.ts

```diff
@@ -8,14 +8,21 @@ export abstract class Fetch {
     const securedHeader: RequestInit = {
       ...options,
       headers: {
-        Accept: 'application/json',
+        'Accept': 'application/json',
         'Content-Type': 'application/json',
         'token': token
       }
     };
 
```

Comment:
You will recive from server all data
200, 404, 500 for example, 
all you need is to use `.then()` and `.catch()`

Reply:
Please tell me in the room what did you mean here.